lib/anycablebility/cli/app.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'logger'
+require 'anycablebility/rpc'
+require 'anycablebility/tests'
+require 'anycablebility/client_factory'
+
+module Anycablebility
+  module Cli
+    # This class is the entry point of Anycablebility
+    class App
+      def run(
+        target: 'ws://0.0.0.0:8080/cable',
+        redis: 'redis://localhost:6379',
+        debug: false
+      )
+        logger = Logger.new(STDOUT)
+        logger.level = debug ? Logger::DEBUG : Logger::WARN
+
+        rpc = Rpc.instance.configure(redis, debug, logger)
```

Comment:
>     # helper method
>     # later we'll be able to add a colorising here, depending on the level
>     def log(level, message)
>       return unless logger
>       logger.send(level.to_sym, PREFIX) { message }
>     end

There's a `Logger#formatter=`. Maybe we should take advantage of it for that purpose?